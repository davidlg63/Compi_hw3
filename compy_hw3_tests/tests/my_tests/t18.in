int a(){int f;}

void main(){
    auto c = 1;
    while(c){
        continue;
        break;
        break;
        continue;
        break;
        break;
    }
}