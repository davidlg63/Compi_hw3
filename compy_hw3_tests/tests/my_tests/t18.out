---end scope---
f INT 0
line 5: type mismatch
